index.js

```diff
@@ -26,13 +26,13 @@ var appServer = function(config) {
     app_dir: 'apps'
   };
 
-  self.config = defaults(config, defaultOptions);
+  self.config = utils.defaults(config, defaultOptions);
```

Comment:
This is a wrapper for a `require`, seems unnecessary as it provides no additional value.